nupic/research/frameworks/dendrites/metrics/metrics.py

```diff
@@ -157,6 +157,44 @@ def dendrite_activations_by_unit(dendrite_activations, winning_mask, targets):
         return selected_activations
 
 
+def hidden_activations_by_unit(activations, targets):
+    """
+    Returns a 2D torch tensor with shape (num_categories, num_units) where cell c, i
+    gives the mean value (post-sigmoid) of the selected dendrite activation for unit i
```

Comment:
When you say, "post-sigmoid", does this mean the input to this function comes from the output of a sigmoid?

Reply:
I think including this term is erroneous, since I mostly copied the description from `dendrite_activations_by_unit`. This has nothing to do with sigmoids, and I'll update the docstring.

The input to this function `activations` is just the output of a forward module (intended to be a dendritic layer or fc layer) over a batch of inputs. The parameter description for `activations` is correct.

Reply:
Updated.